TelCo.ColorCoder/GetPair.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+
+namespace TelCo.ColorCoder
+{
+    public class GetPair
+    {
+
+        /// <summary>
+        /// Given the two colors the function returns the pair number corresponding to them
+        /// </summary>
+        /// <param name="pair">Color pair with major and minor color</param>
+        /// <returns></returns>
+        public static int GetPairNumberFromColor(PairsDataModel.ColorPair pair)
+        {
+            // Find the major color in the array and get the index
+            int majorIndex = -1;
+            for (int i = 0; i < PairsDataModel.colorMapMajor.Length; i++)
+            {
+                if (PairsDataModel.colorMapMajor[i] == pair.majorColor)
+                {
+                    majorIndex = i;
+                    break;
+                }
+            }
+
+            // Find the minor color in the array and get the index
+            int minorIndex = -1;
+            for (int i = 0; i < PairsDataModel.colorMapMinor.Length; i++)

```

Comment:
This `for` loop is almost identical to the previous one - only difference is the array being searched.
Reduce the duplication

Reply:
Updated for loop code to reduce duplication.

Reply:
Nice work :)